### DISCUSSION TITLE
How long will it take for a renamed plugin to show up on the list?

### DISCUSSION THREAD

**Discussion by jeffwright13** [pytest-dev/pytest#9899] [General] [answered]
I had a plugin named `pytest-fold` that was on the official plugin list. I renamed the repo (using the cookie cutter method). The plugin list no longer shows the old name, which is good - but it also doesn't yet show the new name (`pytest-tui`). 

How long can I expect to wait before it gets up on that list?

**Comment by The-Compiler** [comment, marked as answer]
The plugin list update runs weekly via a GitHub Action: https://github.com/pytest-dev/pytest/actions/workflows/update-plugin-list.yml

Thus, you should see it updated somewhere around Monday (IIRC the PR still gets merged manually).

**Reply by jeffwright13** [reply]
Thanks for the response. The new repo has been on GH now for a few weeks, but I'll wait until after Monday to see if maybe it gets picked up this time.

**Reply by nicoddemus** [reply]
Just to clarify that the action lists packages on PyPI, not on GitHub.

**Reply by jeffwright13** [reply]
Yes, thanks. It's on PyPi.

**Reply by The-Compiler** [reply]
The package [is in the plugin list](https://docs.pytest.org/en/latest/reference/plugin_list.html) already (note that you will need to view the "latest" docs).

**Reply by jeffwright13** [reply]
Cool! I was looking [here](https://docs.pytest.org/en/7.0.x/reference/plugin_list.html). Thanks for the reply - I guess I'm pretty much super-famous now for having a Pytest plugin advertised on the official list.
